src/components/site-menu.tsx

```diff
@@ -100,9 +100,9 @@ function SiteItem( { site }: { site: SiteDetails } ) {
 	return (
 		<li
 			className={ cx(
-				'flex flex-row min-w-[168px] h-8 hover:bg-[#ffffff0C] focus:bg-[#ffffff0C] rounded transition-all',
```

Comment:
The focus styles applied to the `li` element did not appear to have an impact, as the element is not focusable. The child button is the element focused when moving focus with the keyboard, which has its own focus styles with an outline.